wagtail/search/backends/database/mysql/query.py

```diff
@@ -235,10 +234,13 @@ def __init__(
     ) -> None:
         super().__init__(output_field=output_field)
         self.query = query
-        self.columns = columns or [
-            "title",
-            "body",
-        ]  # We need to provide a default list of columns if the user doesn't specify one. We have a joint index for for 'title' and 'body' (see wagtail.search.migrations.0006_customise_indexentry), so we'll pick that one.
+        self.columns = (
```

Comment:
@laymonage now I remember why I manually formatted this previously. With the trailing comment it wraps it all in `()` and adds to the multi-line goofiness when this could easily be a one-liner if the comment was placed above the code rather than inline. In the interests of getting this merged, I'll keep my current PR focused on the automatic `ruff format` changes and leave the manual cleanup for another day.